### PR TITLE
fix: remove afk screen

### PR DESCRIPTION
**Pull request by pollend**
removed the afk screen and moved the console messages to logging instead. having an popup show up every minute or so is kind of obnoxious. probably just enough to mark the player as afk rather then also notify them that they are afk. I don't think there is really a need to convey this to player in the first place. in single player this popup shows up constantly so that needs to be addressed as well. it would need to be disabled in singe player since afk is not really relevant in this case. mostly to signal other players that the given player is not available to interact with. 

ref: https://github.com/MovingBlocks/Terasology/pull/3571

### CODE REVIEW

File: engine/src/main/java/org/terasology/logic/afk/AfkClientSystem.java [hunk 1]
```diff
@@ -77,14 +79,11 @@ public void onCommand() {
         component.afk = !component.afk;
         entity.addOrSaveComponent(component);
         if (component.afk) {
-            nuiManager.pushScreen(SCREEN_URL, AfkScreen.class).setAfkClientSystem(this);
-            nuiManager.closeScreen(CONSOLE_SCREEN_URL);
             enableDiscord();
-            console.addMessage("[AFK] You are AFK now!");
+            logger.info("[AFK] You are AFK now!");
```
Comment (skaldarnar) [inline engine/src/main/java/org/terasology/logic/afk/AfkClientSystem.java:83]: I think some form of confirmation would be nice... Console is the text box in the bottom left, right? I think we could also print to the dev console by returning a String from this command method.

File: engine/src/main/java/org/terasology/logic/afk/AfkClientSystem.java [hunk 2]
```diff
@@ -77,14 +79,11 @@ public void onCommand() {
...
         } else {
-            nuiManager.closeScreen(SCREEN_URL);
             disableDiscord();
-            console.addMessage("[AFK] You are no longer AFK!");
+            logger.info("[AFK] You are no longer AFK!");
```
Comment (skaldarnar) [inline engine/src/main/java/org/terasology/logic/afk/AfkClientSystem.java:86]: Same here, application logs are not visible to the player.

**Review by skaldarnar — CHANGES_REQUESTED** [review]

**Review by skaldarnar — APPROVED** [review]

**Comment by jdrueckert** [comment]
Closes #4243

**Comment by pollend** [comment]
is there any other changes we want to make @skaldarnar.

**Review by skaldarnar — APPROVED** [review]

**Pull request merged**